meta/main.yml

```diff
@@ -0,0 +1,30 @@
+galaxy_info:
+  author: Team Starship
```

Comment:
I think we should also remove the company line, given that the original author doesn't work for EBSCO and if they wanted to merge our changes so they could publish to Ansible Galaxy, I don't think it would make sense for EBSCO to be attached